### DISCUSSION TITLE
why this git repo so big?

### DISCUSSION THREAD

**Discussion by meritozh** [saadeghi/daisyui#2440] [General]
1.41GiB repo size. what's the reason?
![image](https://github.com/saadeghi/daisyui/assets/7543300/3a6d6a89-090b-49cd-861f-6056f7aab701)

**Comment by NikarashiHatsu** [comment]
This repo contains the library itself and the documentation page. Try to clone the repo using the `--depth=1` flag, it should decrease the size a bit.

**Comment by saadeghi** [comment]
You can also fork the repo and only fork the master branch. This would be much smaller

![Screenshot_2023-10-24-15-11-46-329_com android chrome](https://github.com/saadeghi/daisyui/assets/7342023/2f35996a-d157-4878-bcef-7addb44905b3)
